# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../scons/environment.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We must inherit the build environment to have the PATH to the wrappers as well
as their configuration.

  - Rene Rebe <rene@exactcode.de>

--- ./engine/SCons/Action.py.orig	2005-09-08 09:15:13.000000000 -0400
+++ ./engine/SCons/Action.py	2005-11-12 02:19:01.160181888 -0300
@@ -437,7 +437,7 @@
         escape = env.get('ESCAPE', lambda x: x)
 
         try:
-            ENV = env['ENV']
+            ENV = os.environ
         except KeyError:
             global default_ENV
             if not default_ENV:
